code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -176,16 +176,14 @@ stare
 		for(var/turf/simulated/floor/feather/T in view(max_dist, holder.owner))
 			return FALSE
 
-
 /datum/aiTask/sequence/goalbased/nest/get_targets()
 	. = list()
 	var/mob/living/critter/flock/F = holder.owner
-
 	for(var/turf/simulated/floor/T in view(max_dist, holder.owner))
-		if(!isfeathertile(T))
-			if(F?.flock && !F.flock.isTurfFree(T, F.real_name))
-				continue
-			. += T
+		if(F?.flock && !F.flock.isTurfFree(T, F.real_name))
+			continue
+		. += T
+		break
```

Comment:
shouldn't this path check before breaking (like the other tasks), otherwise it'll break on the first valid tile found regardless of whether it can get to it?

Reply:
We can add it. It's probably a rare occurrence since in most cases the closest simulated turf is the one you are already standing on. If somehow a drone ends up in space I don't believe the path finding will put them back on station anyway. So really that just leaves the arrivals and escape shuttles unless admins are making unsimmed areas. At any rate it will probably only add minimal time anyway